nix/process-compose-flake-lib.nix

```diff
@@ -0,0 +1,99 @@
+{ lib }:
+let
+  inherit (lib) types;
+in
+{
+  mkProcessComposeArgumentsOption = {}:
```

Comment:
Why is this a function, instead of a simple assignment to the option?

Reply:
I followed convention here, (mkOption, mkPerSystemOption), also it could later be extended to take arguments this way, for example to set default, without breaking code. But I can change it if you want.